### PR TITLE
Add firefox/set-as-default/ .ftl files

### PR DESCRIPTION
**Pull request by alexgibson**
Ref: https://github.com/mozilla/bedrock/pull/9027

### CODE REVIEW

File: en/firefox/set-as-default/landing.ftl [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+set-as-default-landing-firefox-is-fast-and = { -brand-name-firefox } is fast and safe on { -brand-name-windows }, { -brand-name-ios }, { -brand-name-android }, { -brand-name-linux }…and across them all. You deserve choices in browsers and devices, instead of decisions made for you.
+set-as-default-landing-choose-corporate-independence = Choose corporate independence
+set-as-default-landing-firefox-is-the-only = { -brand-name-firefox } is the only major independent browser. { -brand-name-chrome }, { -brand-name-edge } and { -brand-name-brave } are all built on { -brand-name-google } code, which means giving { -brand-name-google } even more control of the internet.
+set-as-default-landing-the-internet-keeps = The internet keeps finding new ways to poach your personal data. { -brand-name-firefox } is the only browser with a mission of finding new ways to protect you.
+set-as-default-landing-firefox-is-fast-no-interest = { -brand-name-firefox } is fast and safe on { -brand-name-windows }, { -brand-name-ios }, { -brand-name-android }, { -brand-name-linux }...and across them all. We have no interest in locking you in or resetting your preferences.
```
Comment (peiying2) [inline en/firefox/set-as-default/landing.ftl:22]: Nit pick: use `…` instead of `. . .`

**Comment by alexgibson** [comment]
@peiying2 thank you updated

**Comment by peiying2** [comment]
> @peiying2 thank you updated

Sorry, I forgot en-GB, en-CA. Could you run another update? Thank you.

**Comment by alexgibson** [comment]
ah good catch! done

**Pull request merged**